back-end/calender/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from basemodel.models import BaseModel
+
+
+# Create your models here
+
+class Calender(BaseModel):
```

Comment:
Spell error (correct -> Calendar). Will cause a lot of confusion in the future if not corrected.

Reply:
I will correct it in next commit